packages/tldraw/src/lib/shapes/text/TextShapeUtil.tsx

```diff
@@ -141,6 +142,8 @@ export class TextShapeUtil extends ShapeUtil<TLTextShape> {
 
 	indicator(shape: TLTextShape) {
 		const bounds = this.editor.getShapeGeometry(shape).bounds
+		const editor = useEditor()
+		if (shape.props.autoSize && editor.editingShapeId === shape.id) return null
```

Comment:
Don't show indicator when editing an auto-size text shape.
We still show it when you're not editing.